### PR TITLE
jquery.mockjax

### PR DESCRIPTION
**Pull request by mplatings**
This is slightly modified from https://github.com/jakerella/jquery-mockjax/issues/204

### CODE REVIEW

File: jquery.mockjax/jquery.mockjax.d.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+interface MockJaxStatic {
+    (options: MockJaxSettings): number;
+    mockedAjaxCalls(): any; // TODO: any?
+    unfiredHandlers(): any; // TODO: any?
+}
+
+interface JQueryStatic {
+    mockjax: MockJaxStatic;
+    mockjaxClear(id?: number): any;
```
Comment (jakerella) [inline jquery.mockjax/jquery.mockjax.d.ts]: This is a deprecated function and will be removed in v2.0.0! You should use `$.mockjax.clear()` instead.

File: jquery.mockjax/jquery.mockjax.d.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    responseText?: string;
+    responseXml?: string;
+    proxy?: string;
+    lastModified?: string;
+    etag?: string;
+}
+
+interface MockJaxStatic {
+    (options: MockJaxSettings): number;
+    mockedAjaxCalls(): any; // TODO: any?
```
Comment (jakerella) [inline jquery.mockjax/jquery.mockjax.d.ts]: There's also an `unmockedAjaxCalls()` method now.

**Comment by jakerella** [comment]
We need to add [`handler()`](https://github.com/jakerella/jquery-mockjax/blob/master/jquery.mockjax.js#L671) as well, for retrieving a single mock handler.

**Comment by vvakame** [comment]
@mplatings thank you your contributions.
please check this document. http://definitelytyped.org/guides/contributing.html

* [ ] Naming the file. 
  * https://www.npmjs.com/package/jquery-mockjax
  * rename jquery.mockjax/jquery.mockjax.d.ts to jquery-mockjax/jquery-mockjax.d.ts
* [ ] Tests

**Comment by vladeck** [comment]
Has anything been done about the inclusion of mockjax typescript definition? if not, what steps are remaining so I can make this happen?

**Comment by vvakame** [comment]
I think you should make another PR.

**Comment by vvakame** [comment]
#7356

**Comment by RyanCavanaugh** [comment]
Hello and thank you for your contribution!

Due to an excessively long queue of pull requests that have become stale, we are "declaring bankruptcy" and closing all PRs opened before May 1, 2016. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

Going forward, we are committing to review or merge all PRs on a regular basis so this bankruptcy will not occur again. We apologize for the incovenience and hope you will continue to contribute to DefinitelyTyped in the future.

Thanks
The TypeScript and DefinitelyTyped teams

**Pull request closed without merging**